db/migrate/20230608071425_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :name
```

Comment:
Why this field is added ?
comment - name ?

Reply:
Sir, name field shows the name of the commentor